aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -7,6 +7,13 @@
 #include <ATen/native/mps/OperationUtils.h>
 #include <torch/library.h>
 
+// TODO: Remove me when moved to MacOS 13
+@interface MPSGraph (VenturaOps)
```

Comment:
I think we should probably move this to a header file and include all the venturaOps in the same place.